automerge-backend-wasm/scripts/js_tests.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+if [[ -e "../../automerge" ]]
+then
+    echo "Automerge repo exists. Skipping cloning..."
+else
+    echo "Cloning automerge repo to ../../automerge"
+    git clone https://github.com/automerge/automerge.git ../../automerge
+fi
+
+ORIGINAL_PWD=$PWD
+cd ../../automerge
+git checkout 81079ff75d2234b47cb912bad728158f2e71c527
+yarn install
+env WASM_BACKEND_PATH="$ORIGINAL_PWD/build" yarn testwasm
```

Comment:
This doesn't seem to build the backend at all so with just running `yarn test:js` I get an `Error: Cannot find module '/home/andrew/projects/automerge-rs/automerge-backend-wasm/build'`. I'd expect this to handle the dependencies of the task (like cloning automerge) including building the wasm backend.

Reply:
The reason I don't build the wasm backend is b/c sometimes we might want to build dev & sometimes prod (depending on how much time you have, etc.) so I left that up to the user. (It's 1 command: `yarn run build` or `yarn run release`).

Does this seem reasonable?

Reply:
Going to temporary resolve to trigger CI. (Not sure if unresolved issue is preventing CI from running?)

Reply:
I think the best thing we can do to ensure interoperability is have interop testing as part of CI. In that context I think we do need to automate the build step. 

I think the environment variable approach o alluded to earlier should help here for cases where the user wants a different build?

Reply:
In CI (inside .travis.yml), I do:

```
cd automerge-backend-wasm && yarn build && yarn test:js
```

To build & then test js interop with the assumption that locally users can run `yarn build` or `yarn release` themselves since they want flexibility while developing. Should I also handle building while developing.

Reply:
I'm happy to have the users do a `yarn build` or equivalent. A friendlier error message would be nice, e.g. if the `build` directory is missing then ask them to build, giving them an example of build or release.

Reply:
done